apps/angular-store-app/src/app/posts/state/post.effects.ts

```diff
@@ -0,0 +1,12 @@
+import { Injectable } from '@angular/core';
+import { Actions, Effect } from '@datorama/akita-ng-effects';
+import { PostsStore } from './posts.store';
+import { tap } from 'rxjs/operators';
```

Comment:
Both modules are lazy-loaded. I'd test it on eager modules.

Reply:
I also added for feature effects in eager modules.